onedal/tests/utils/_dataframes_support.py

```diff
@@ -119,6 +120,8 @@ def _as_numpy(obj, *args, **kwargs):
         return dpt.to_numpy(obj, *args, **kwargs)
     if isinstance(obj, pd.DataFrame) or isinstance(obj, pd.Series):
         return obj.to_array(*args, **kwargs)
+    if sp.issparse(obj):
+        return obj.toarray(*args, **kwargs)
```

Comment:
Is it really needed in the scope of the PR? Please remove not used or not needed code from the PR.

Reply:
Please see the results of kneighbors_graph, there are circumstances where a scipy csr_matrix is returned. This is a necessary addition.